support-frontend/assets/helpers/productCatalog.tsx

```diff
@@ -120,49 +143,59 @@ export const productCatalogDescription: Record<string, ProductDescription> = {
 		ratePlans: {
 			Monthly: {
 				billingPeriod: 'Monthly',
+				deliverableTo: gwDeliverableCountries,
```

Comment:
Does this country list need to be split between GuardianWeeklyDomestic & GuardianWeeklyRestOfWorld?